### ISSUE TITLE
privateKey in ~/.litconfig is ignored

### ISSUE THREAD

**Issue by ghost** [luvit/lit#139]
```
$ grep privateKey /home/AD/tf128/.litconfig
privateKey: ${HOME}/.ssh/id_rsa_4096
$ lit auth urzds | grep -e config -e privateKey
load config: ${HOME}/.litconfig
privateKey: ${HOME}/.ssh/id_rsa
```

This seems wrong, as it contradicts https://github.com/luvit/lit/wiki/Creating-Your-First-Package#authentication (3rd paragraph).

(In the output above, `${HOME}` is the actual path, not the envvar reference.)

**Comment by squeek502** [comment]
Yep, looks like [`${HOME}/.ssh/id_rsa` overwrites the config setting if the path exists](https://github.com/luvit/lit/blob/ca24ebaf65b1a59dc87ea323e125804c96ad22ff/commands/auth.lua#L54-L57).

**Comment by creationix** [comment]
Let me know if this patch helps.
